development/nautobot_config.py

```diff
@@ -83,17 +73,16 @@ def is_truthy(arg):
 CACHES = {
     "default": {
         "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": f"{REDIS_SCHEME}://{REDIS_HOST}:{REDIS_PORT}/0",
+        "LOCATION": parse_redis_connection(redis_database=0),
         "TIMEOUT": 300,
         "OPTIONS": {
             "CLIENT_CLASS": "django_redis.client.DefaultClient",
-            "PASSWORD": REDIS_PASSWORD,
         },
     }
 }
 
 # REDIS CACHEOPS
-CACHEOPS_REDIS = f"{REDIS_SCHEME}://:{REDIS_PASSWORD}@{REDIS_HOST}:{REDIS_PORT}/1"
+CACHEOPS_REDIS = parse_redis_connection(redis_database=1)
```

Comment:
Not blocking for this PR but it would be good to revisit nautobot_config.py as it looks like it's still got a *lot* of settings below that shouldn't be needed or relevant for app development.